pkgs/development/python-modules/HPI/default.nix

```diff
@@ -0,0 +1,47 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, appdirs
+, click
+, decorator
+, logzero
+, lxml
+, more-itertools
+, mypy
+, orjson
+, pandas
+, pytz
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "HPI";
+  version = "0.0.20200417";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-cozMmfBF7D1qCZFjf48wRQaeN4MhdHAAxS8tGp/krK8=";
+  };
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  propagatedBuildInputs = [
+    appdirs
+    click
+    decorator
+    logzero
+    lxml
+    more-itertools
+    mypy
+    orjson
+    pandas
+    pytz
+  ];
```

Comment:
Optional dependencies should go into `passthru.optional-dependencies.<extraName>`.

https://github.com/karlicoss/HPI/blob/master/setup.py#L56-L62